### ISSUE TITLE
Bug with mpv configs

### ISSUE THREAD

**Issue by tostr7191** [rockymadden/somafm-cli#10]
Nice project, replaces my little manual script file I had set up.

Unfortunately it does not play well with my mpv config, which includes the `pause` option. This options starts anything in paused mode.

I could see two ways of solving this, either add `--pause=no` to the mpv execution, or add the `--no-config` option. Probably just overwriting pause is the least intrusive change for the end user.

Thanks and best
t0str

**Comment by rockymadden** [comment]
Greetings! Happy to help where I can.

I'm hesitant to do `--pause=no` for the potential of having to support other settings folks have in their config but need tweaked (since the default is `no`). I tried to confirm `--no-config` usage and I am not given an error, however I came across this post that seems like it might no longer be supported: https://github.com/mpv-player/mpv/issues/555

Any insight?
